### PR TITLE
fix: discord on command + k dark mode and fixing the link

### LINKED ISSUE #2100: cmd+k discord icon in darkmode
### Preliminary Checks

- [X] I have reviewed the documentation: https://unkey.com/docs

- [X] I have searched for existing issues: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Reproduction / Replay Link (Optional)

_No response_

### Issue Summary

The icon color is off on darkmode
image

### Steps to Reproduce

1. go to /settings
2. switch to dark mode
3. press [cmd] + [k]

### Expected behavior

The icons should be bright

### Other information

_No response_

### Screenshots

_No response_

### Version info

```markdown
- OS:
- Node:
- npm:
```

### PR DESCRIPTION
**Pull request by harshsbhat**
## What does this PR do?

Fixes #2100 

The Discord icon in the Cmd + k is displayed correctly in both dark and light mode

Before hover:
image

On hover:
image

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [Contributing Guide](./CONTRIBUTING.md)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand areas
- [x] Ran `pnpm build`
- [x] Ran `pnpm fmt`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

## Summary by CodeRabbit

- **New Features**
	- Updated the `CommandItem` in the Command Menu to navigate to an external link for Discord.

- **Style**
	- Enhanced visual presentation of the Command Menu with improved SVG styling.

### CODE REVIEW

File: apps/dashboard/components/dashboard/command-menu.tsx [hunk 1]
```diff
@@ -68,9 +68,9 @@ export function CommandMenu() {
 const DiscordCommand: React.FC = () => {
   const router = useRouter();
   return (
-    <CommandItem onSelect={() => router.push("/discord")}>
+    <CommandItem onSelect={() => router.push("https://unkey.com/discord")}>
       <svg
-        className="w-4 h-4 mr-2"
+        className="w-4 h-4 mr-2 text-zinc-500 hover:text-white dark:text-zinc-400 dark:hover:text-white transition-colors duration-200"
```
Comment (chronark) [inline apps/dashboard/components/dashboard/command-menu.tsx:73]: I think this should be enough to inherit the correct colors
```suggestion
        className="w-4 h-4 mr-2 text-zinc-500 hover:text-white dark:text-zinc-400 dark:hover:text-white transition-colors duration-200"
        className="w-4 h-4 mr-2 fill-current"
```

File: apps/dashboard/components/dashboard/command-menu.tsx [hunk 2]
```diff
@@ -79,7 +79,8 @@ const DiscordCommand: React.FC = () => {
...
-            fill-rule="nonzero"
+            fill="currentColor"
```
Comment (chronark) [inline apps/dashboard/components/dashboard/command-menu.tsx:82]: no need to set fill here and we can remove fillRule as well

**Review by chronark — CHANGES_REQUESTED** [review]

File: apps/dashboard/components/dashboard/command-menu.tsx (hunk 2 above)
Comment (harshsbhat) [reply apps/dashboard/components/dashboard/command-menu.tsx]: done

File: apps/dashboard/components/dashboard/command-menu.tsx [hunk 3]
```diff
@@ -79,7 +79,6 @@ const DiscordCommand: React.FC = () => {
...
-            fill="currentColor"
             fillRule="nonzero"
```
Comment (chronark) [inline apps/dashboard/components/dashboard/command-menu.tsx:82]: can you please remove this too

**Review by chronark — CHANGES_REQUESTED** [review]

File: apps/dashboard/components/dashboard/command-menu.tsx (hunk 3 above)
Comment (harshsbhat) [reply apps/dashboard/components/dashboard/command-menu.tsx]: I am sorry. My bad. I don't know how I missed that.

**Review by chronark — APPROVED** [review]

**Pull request merged**